### DISCUSSION TITLE
folding (of ts files) through WildWebDeveloper constantly redraws/relayouts the editor

### DISCUSSION THREAD

**Discussion by jcompagner** [mickaelistria/eclipse-bluesky#1517] [General]
not sure where the problem is is it somehow in the language service or is it T4ME?
so i asked also the question here:

https://github.com/eclipse/tm4e/discussions/742

**Comment by mickaelistria** [comment]
I think @sebthom nailed it, it seems like an instance of https://github.com/eclipse/lsp4e/issues/895

**Comment by nelson777** [comment]
I'm experiencing the same problem. Is there any workaround yet ? This is really annoying.

**Comment by MrsNickalo** [comment]
I'm experiencing this issue and it is driving me crazy! Is there a fix for this?

**Reply by mickaelistria** [reply]
Not that we're aware of. Contributions are welcome!

**Comment by MoonMao42** [comment]
Clips are not automatically purged after a set time. They stay in history until you either clear them manually (Edit > Clear) or until the history hits the max size limit you set in Settings > Storage (up to 999). Once the limit is reached, oldest entries get removed as new ones come in.

If your admin needs data wiped on logout or something like that, you could set up a script to delete ~/Library/Containers/org.p0deje.Maccy (or the relevant plist) on a schedule. Not ideal but it works.
